health-services/project-factory/src/server/api/campaignApis.ts

```
@@ -778,10 +872,15 @@
   dataFromSheet: any,
   request: any,
   createAndSearchConfig: any,
+  properties: any,
   localizationMap?: { [key: string]: string }
 ) {
   try {
     validateEmptyActive(dataFromSheet, request?.body?.ResourceDetails?.type, localizationMap);
+    const errorsRelatedToMultiSelect:any = validateMultiSelect(dataFromSheet, properties, localizationMap);
+    request.body.sheetErrorDetails = request?.body?.sheetErrorDetails
+      ? [...request?.body?.sheetErrorDetails, ...errorsRelatedToMultiSelect]
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)